main.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/yannh/kubeconform/cmd/kubeconform"
+	"github.com/yannh/kubeconform/pkg/config"
+)
+
+var version = "development"
+
+func main() {
+	cfg, out, err := config.FromFlags(os.Args[0], os.Args[1:])
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "failed parsing command line: %s\n", err.Error())
+		os.Exit(1)
+	}
+
+	if err = kubeconform.Validate(cfg, out); err != nil {
+		fmt.Fprintf(os.Stderr, "failed validating resources: %s - %s\n", err.Error(), out)
```

Comment:
this is incorrect.. kubeconform currently returns 1 when some resources fail to validate - we need to differentiate errors processing from errors when validating.. 
It could return a bool saying if everything was valid or not?

Reply:
I didn't change the logic here, I just used what's there but with an error instead of an init.
So any `return 1` became a `return error` and any `return 0` became a `return nil`.